example/bosh-lite-manifest-template.yml

```diff
@@ -25,149 +29,348 @@ update:
   max_in_flight: 3
   update_watch_time: 1000-300000
 
+## Instance Group details ##
 
 instance_groups:
+
+# Consul instance group
+  - name: consul
+    azs:
+      - z1
+    instances: 1
+    update:
+      serial: true
+    stemcell: default
+    vm_type: default
+    networks:
+      - name: default
+        static_ips: &consul_ips
+          (( consul_properties.static_ips ))
+    jobs:
+      - name: consul_agent
+        properties:
+          consul:
+            agent:
+              mode: server
+              domain: cf.internal
+              servers: &consul_servers
+                lan: *consul_ips
+            ca_cert: (( consul_properties.ca_cert ))
+            agent_cert: (( consul_properties.agent_cert ))
+            agent_key: (( consul_properties.agent_key ))
+            server_cert: (( consul_properties.server_cert ))
+            server_key: (( consul_properties.server_key ))
+            encrypt_keys: (( consul_properties.encrypt_keys ))
+        release: consul
+
+# Postgres Instance Group
+
   - name: postgres
     azs:
       - z1
     instances: 1
+    update:
+      serial: true
+    stemcell: default
+    vm_type: default
+    networks:
+      - name: default
     jobs:
+      - name: consul_agent
+        properties:
+          consul:
+            agent:
+              mode: client
+              domain: cf.internal
+              services:
+                postgres:
+                  name: postgres
+                  tags:
+                  - postgres
+                  - db
+                  check:
```

Comment:
No. Postgres isn't HA, so if we had a health check, we would see the DNS entry come and go which would be back for those trying to access it. Right now it will just fail with a cannot connect.
In environments where you know how to deploy an HA Postgres, you can do whatever you want to get the DNS working properly.